docker/push.sh

```diff
@@ -10,7 +10,7 @@ else
   id="${SEMAPHORE_JOB_UUID:0:8}"
 fi
 
-image="renderedtext/hubot:${REVISION:0:8}-${id}"
+export image="renderedtext/hubot:${REVISION:0:8}-${id}"
```

Comment:
Does this setup work? A typical bash script should not be able to modify its environment.

I wasn't able to reproduce it in a test case:

```
➜  ~ cat /tmp/a.sh
export A="111"

➜  ~
➜  ~ bash /tmp/a.sh
➜  ~ echo $A

➜  ~
```

Reply:
Hmm, is there some other way to pass $image value from `push` script to `deploy`?